pkg/ebpf/tracer.go

```diff
@@ -406,6 +406,18 @@ func (t *Tracer) getConnections(active []ConnectionStats) ([]ConnectionStats, ui
 					conn.DPort,
 					process.ConnectionType(conn.Type),
 				)
+
+				if conn.DPort == 53 {
+					key := connKey{
+						serverIP:  conn.Dest,
+						queryIP:   conn.Source,
+						queryPort: conn.SPort,
+						protocol:  conn.Type,
+					}
+					dnsStats := t.reverseDNS.GetDNSStats(key)
```

Comment:
Any reason this can't just take a conn and have it's internal key representation constructed within `GetDNSStats()`?